src/main/java/codeu/model/store/basic/UserStore.java

```diff
@@ -126,5 +127,12 @@ public boolean isUserRegistered(String username) {
   public void setUsers(List<User> users) {
     this.users = users;
   }
+  
+  
+  //JJ Returns number of users.
+  public int numUsers(){
```

Comment:
would name this getNumUsers. Please fix the whitespace to match the rest of the file.